packages/blockchain/test/util.ts

```diff
@@ -55,6 +55,41 @@ export const generateBlockchain = async (numberOfBlocks: number, genesis?: Block
     error: null,
   }
 }
+/**
+ *
+ * @param parentBlock parent block to generate the consecutive block on top of
+ * @param difficultyChangeFactor this integer can be any value, but will only return unique blocks between [-99, 1] (this is due to difficulty calculation). 1 will increase the difficulty, 0 will keep the difficulty constant any any negative number will decrease the difficulty
+ */
+
+export const generateConsecutiveBlock = (
+  parentBlock: Block,
+  difficultyChangeFactor: number
+): Block => {
+  if (difficultyChangeFactor > 1) {
+    difficultyChangeFactor = 1
+  }
+  const common = new Common({ chain: 'mainnet', hardfork: 'muirGlacier' })
+  const tmpHeader = BlockHeader.fromHeaderData({
+    number: new BN(parentBlock.header.number).add(new BN(1)),
+    timestamp: parentBlock.header.timestamp.addn(10 + -difficultyChangeFactor * 9),
+  })
+  const header = BlockHeader.fromHeaderData(
+    {
+      number: new BN(parentBlock.header.number).add(new BN(1)),
```

Comment:
same around here, no need to wrap `new BN()`s. can also do `.addn(1)`

Reply:
Will resolve here, these (should be) resolved in #926